lib/EditableList/EditableListForm.js

```diff
@@ -476,7 +482,7 @@ class EditableListForm extends React.Component {
               <Row end="xs">
                 <Col xs>
                   <Button
-                    disabled={isEditing}
+                    disabled={isEditing || canCreate}
```

Comment:
I would expect the "Add new" button to be _disabled_ when `canCreate` is false. Am I thinking of this backwards? 

To maintain existing functionality, I would expect this to be a new prop with a default value of `true` that you would have to pass in as `false` in order to suppress the button.

Reply:
Thanks, I'm agree with you, you are right, I'm already fixed it.